exonum/src/sandbox/consensus/transactions.rs

```diff
@@ -232,6 +232,27 @@ fn duplicate_tx_in_pool() {
     ));
 }
 
+#[test]
+fn rebroadcast_transactions() {
+    let sandbox = timestamping_sandbox();
+
+    let mut transactions = TimestampingTxGenerator::new(DATA_SIZE)
+        .take(5)
+        .collect::<Vec<_>>();
+
+    transactions.sort_by(|a, b| a.hash().cmp(&b.hash()));
+
+    for tx in &transactions {
+        sandbox.recv(tx);
+    }
+
+    sandbox.recv_rebroadcast();
+
+    for tx in &transactions {
+        sandbox.broadcast(tx)
```

Comment:
Do I understand correctly that this check enforces a particular order the transactions should be broadcast in? Ideally, I'd like to have a method to extract all broadcast messages (e.g., via an `Iterator`), and verify that they as a set correspond to the expected set. I suppose this is not critical, but would look good as an improvement.